src/Symfony/Component/Intl/Languages.php

```diff
@@ -122,7 +127,15 @@ public static function alpha3CodeExists(string $language): bool
      */
     public static function getAlpha3Name(string $language, string $displayLocale = null): string
     {
-        return self::getName(self::getAlpha2Code($language), $displayLocale);
+        try {
+            return self::getName(self::getAlpha2Code($language), $displayLocale);
+        } catch (MissingResourceException $e) {
+            if (3 === \strlen($language)) {
```

Comment:
Please change this line to:
```
if (2 < \strlen($language)) {
```

Reply:
i tend to take further care of this in (or after) #33148 actually, i assume you're hinting the value might be localized e.g. `xxx_XX`. For now the 3-letter case is expected only